tests/check_consistent.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 
-# For security (and simplicity) reasons, only a limited kind of files can be
-# present in /stdlib and /stubs directories, see README for detail. Here we
-# verify these constraints.
+# Check that the typeshed repository contains the correct files in the
+# correct places, and that various configuration files are correct.
```

Comment:
We could make this a docstring while we're here:

```suggestion
"""
Check that the typeshed repository contains the correct files in the
correct places, and that various configuration files are correct.
"""